pom.xml

```diff
@@ -330,7 +330,7 @@
             <groupId>org.slf4j</groupId>
             <artifactId>slf4j-log4j12</artifactId>
           </exclusion>
-        </exclusions>
+      </exclusions>
```

Comment:
Revert this change. (Previous indentation is correct)

Reply:
I'm looking at the file and I don't see anything wrong with the indentation... what is happening?